### ISSUE TITLE
Add important links for each organisation

### ISSUE THREAD

**Issue by nishantwrp** [nishantwrp/gsoc-organizations#81]
Each organisation has a couple of resources for new contributors to get started. Or maybe specific resources for contributors interested in doing a Google Summer of Code project with that organisation. These links are often not present in the official Google Summer of Code website and hence can't be scraped from there. It would be great if we have a section beside each organization showing these important links for that organization. 

I've created [imp_links.json](https://github.com/nishantwrp/gsoc-organizations/blob/aa0fd97c849f9c50d97de36d882b833e887efcd7/api/imp_links.json) that will act as a data source for these links. Please feel free to submit pull requests for any links that you feel are important and are not present in this file. 

PRs for reference: #79, #80

**Comment by srigundeti10** [comment]
@nishantwrp please assign me

**Comment by nishantwrp** [comment]
@srigundeti10, please feel free to create a pull request. Not assigning because I don't want to restrict only assignees to work on this issue.

**Comment by MahendraBishnoi29** [comment]
@nishantwrp can I add links for any organisation?

**Comment by nishantwrp** [comment]
Yes @MahendraBishnoi29, add any link that are relevant to gsoc, beginners looking to contribute to that organization.

**Comment by IshitaPathak** [comment]
Hi @nishantwrp , I've created a pull request for this issue. Could you please take a look and consider merging it? Let me know if there are any changes that need to be made. Thank you!

**Comment by Yadvendra016** [comment]
hey, If this issue is still open please assign me

**Comment by beingnoble03** [comment]
@Yadvendra016 Please create a PR. This issue isn't supposed to be assigned to anyone.

**Comment by Yadvendra016** [comment]
@beingnoble03 can you please explain a little bit more about the issue

**Comment by beingnoble03** [comment]
We are building a feature for Important links of an Organization. So, we need to collect the data manually. You can add important links in imp_links.json

Important links can contains anything useful for contribution in that particular org 

For example, link to their official documentation, github repo, anything you can think of.

**Comment by LordMashh** [comment]
![image](https://github.com/nishantwrp/gsoc-organizations/assets/103451383/0d2b0897-cbbd-428c-b572-594a2d9c669d)
I am facing this issue on merging the pull request

**Comment by harshit4311** [comment]
Is this issue still open? If yes, can I directly start working on this?

**Comment by anishsingh1250** [comment]
Hello, I would like to work on this issue.
